gcalcli/utils.py

```diff
@@ -30,7 +32,7 @@
 
 
 def parse_reminder(rem):
-    matchObj = re.match(r'^(\d+)([wdhm]?)(?:\s+(popup|email|sms))?$', rem)
+    matchObj = re.match(REMINDER_REGEX, rem)
```

Comment:
let's go ahead and switch out this camelCasing here (while you're touching this line).  Can you change this to `match` or `match_obj` (honestly, just `match` seems sufficient to me).